.env.dev

```
@@ -11,7 +11,7 @@ DB_NAME=dev-test
 TYPEORM_LOGGING=error,migration,warn,info
 
 # REDIS
-REDIS_VERSION=6.0.8
+REDIS_VERSION=7
```

I would go back to the older version here as this better matches what we have in the deployed infra
```suggestion
REDIS_VERSION=6.0.8
```